docs/blog/posts/announcing-keptn-v2-release-candidate.md

```diff
@@ -0,0 +1,148 @@
+[Multi-stage application delivery](../../docs/guides/multi-stage-application-delivery.md)
+guide.
+
+### Pass OpenTelemetry Contexts through Keptn
+
+Usually, Keptn does not act in isolation during a deployment.
+In the logical timeline of events, there will be tools acting prior to
+Keptn (such as GitOps tools like [ArgoCD](https://argoproj.github.io/cd/)) and tools that act after
+Keptn (such as security scanning tools or post-deployment checks).
+
+If the tools prior to Keptn begin generating OpenTelemetry data (e.g. Spans and traces),
+it would be beneficial to see "Keptn’s portion" of the work in the correct context
+(i.e. as part of the same distributed trace).
+Similarly, anything that happens after Keptn (but still in the same logical "deployment" operation)
+should be included in the end-to-end trace / timing view.
+To achieve this, Keptn now accepts tools to pass [W3C trace IDs](https://www.w3.org/TR/trace-context/) into Keptn
+and Keptn will pass W3C trace IDs out of Keptn
+(thanks to all contributors working on this functionality!)
+Now, you can see a true end-to-end picture of everything in the logical order,
+potentially from "PR merged" all the way to "deployment complete".
+
+![dev-to-proc](./announcing-keptn-v2-release-candidate/dev-to-prod.png)
+
+## How to get started
+
+To get started with Keptn, head over to the [Keptn installation instructions](../../docs/installation/index.md).
+After installing Keptn, you can get started by going through the several guides listed below that
+will introduce you to the features provided by Keptn step by step.
+
+- [Integrate Keptn with your Applications](../../docs/guides/integrate.md):
+This shows you how to configure your applications to be deployed with Keptn.
+- [Deployment tasks](../../docs/guides/tasks.md):
+This builds up on the previous guide and will explain in detail how you can enhance your application
+deployments by adding pre- and post-deployment tasks to ensure you deploy into a healthy environment.
+- [Evaluations](../../docs/guides/evaluations.md):
+This shows you how to use Keptn to evaluate metrics relevant for an application as part of
+the pre- and post-deployment checks.
+- [OpenTelemetry Observability](../../docs/guides/otel.md):
+This shows how to configure Keptn for making use of its observability features, so you
+have a holistic overview of your application deployments and any issues that arise during deployment.
+- [Multi-stage Application Delivery](../../docs/guides/multi-stage-application-delivery.md):
+This guide serves as an example of how you can integrate Keptn with ArgoCD and GitHub to
+take the previous concepts and apply them across multiple application environments,
+giving you full observability of an application deployment from development into production.
+
+Going through these guides will give you a good initial overview of Keptn,
+but be sure to also check out the [additional guides](../../docs/guides/index.md).
```

Comment:
What about `use-cases`?  I'm not actually sure that we need this sentence.  The links get them into some basic information.  If they can't figure out to look around at the rest of the documentation for more information, are they too stupid to be running Keptn?